videomaker/sass/_links.scss

```
@@ -1,4 +1,4 @@
 a:active {
 	background: var(--wp--custom--color--foreground);
```

I wonder if we should add the !important to this line too, otherwise it's possible that one declaration could work and the other could not, leading to the same problem again. What do you think?
```suggestion
	background: var(--wp--custom--color--foreground) !important;
```

---

I also thought this actually! I didn't think it was as likely that a background would be added to active links from GB, but it's definitely possible, and then we'd have the same problem again like you say. Maybe it's best to add it, and then we're doing our best to ensure contrasting colours here.